implants/lib/eldritch/src/assets/copy_impl.rs

```diff
@@ -61,17 +63,23 @@ pub fn copy(starlark_eval: &Evaluator<'_, '_>, src: String, dst: String) -> Resu
 
         if tmp_list.contains(&src_value.to_value()) {
             let env = Environment::from_extra(starlark_eval.extra)?;
-            let file_reciever = env.request_file(src)?;
+            let (tx, rx) = channel();
+            env.send(FetchAssetMessage { name: src, tx })?;
```

Comment:
`asset.copy` vs `FetchAsset` lots of shmoopin' if you ask me

Reply:
/ out of my hands, we want the gRPC API to be `fetch_asset`